src/puppet-padplus.ts

```diff
@@ -380,15 +380,15 @@ export class PuppetPadplus extends Puppet {
       if (searchContact === null) {
```

Comment:
In the new version of padplus, this result will not return null, let's check it again, and do some changes for both related methods.